main/src/main/java/com/sedmelluq/discord/lavaplayer/track/TrackMarkerTracker.java

```diff
@@ -43,9 +68,7 @@ public TrackMarker remove() {
      * @param state The state of the marker to pass to the handler.
      */
     public void trigger(TrackMarkerHandler.MarkerState state) {
```

Comment:
This function needs its documentation updating to state that all registered markers will be triggered with the provided state